app/helpers/application_helper.rb

```diff
@@ -1,2 +1,19 @@
 module ApplicationHelper
+  IMAGE_ASSET_REGEX = /(?<=\/images\/).+\.(png|jpg|svg|gif)$/
+
+  # @return [Hash<String, String>] A key-value pair of the image assets coming from the Rails
+  #                                asset pipeline.
+  def image_assets
```

Comment:
This is a helper so that we can reference our images in JS. It'll take the path and name of the image file javascript-ify it as the key reference to url asset. So `icons/brand-offer.png` will turn into `iconsBrandOfferPng`.